### PR TITLE
feat: single search session query

### PR DESCRIPTION
**Pull request by sshanzel**
Fetch a single search session by session id from the session history list.

WT-1650 #done

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]
Looks good

**Review by idoshamun — APPROVED** [review]

File: src/schema/search.ts [hunk 1]
```diff
@@ -21,11 +23,45 @@ export const typeDefs = /* GraphQL */ `
...
+    url: String!
+  }
+
+  type SearchChunk {
+    id: String!
+    prompt: String!
+    response: String!
+    error: SearchChunkError
+    createdAt: String!
+    completedAt: String!
```
Comment (idoshamun) [inline src/schema/search.ts:44]: Should be date

**Review by idoshamun — CHANGES_REQUESTED** [review]
Check my comment

File: src/integrations/magni.ts [hunk 2]
```diff
@@ -61,3 +61,47 @@ export const getSessions = async (
...
+  userId: string,
+  sessionId: string,
+): Promise<Search> => {
+  const url = `${magniOrigin}/sessions?id=${sessionId}`;
+  const res = await fetch(url, {
+    ...fetchOptions,
+    headers: { 'X-User-Id': userId },
+  });
+
+  if (!res.ok) throw new ValidationError(await res.text());
```
[the remark this answers is not in the document]
Comment (sshanzel) [reply src/integrations/magni.ts:104]: [After some discussion](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/584286210/COM-90+AI+Search?focusedCommentId=596836378), the error that we would parse would only be coming from successful requests `res.ok = true`.

**Review by idoshamun — APPROVED** [review]

**Pull request merged**